Library/Formula/mergepbx.rb

```diff
@@ -0,0 +1,11 @@
+class Mergepbx < Formula
+  homepage "https://github.com/simonwagner/mergepbx"
+  url "https://github.com/simonwagner/mergepbx/archive/v0.9.tar.gz"
+  version "0.9"
+  sha256 "d0d859a902357f8059fca0c5fb061a9072b82cb7cce4f38bd9eb2c3198fce003"
+
+  def install
+    system "./build.py"
+    bin.install "mergepbx"
+  end  
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!